CoreScriptsRoot/ServerCoreScripts/ServerChat/Speaker.lua

```diff
@@ -0,0 +1,187 @@
+local source = [[
+local module = {}
+
+--////////////////////////////// Details
+--//////////////////////////////////////
+local metatable = {}
+metatable.__ClassName = "Speaker"
+
+metatable.__tostring = function(tbl)
```

Comment:
Seems like we do this metatable business a lot, could we make a generic `MakeClass` function? Or something to that extent?